vmdb/config/initializers/fast_gettext.rb

```diff
@@ -18,17 +18,27 @@ def fix_i18n_available_locales
   I18n.available_locales += FastGettext.available_locales.grep(/_/).map { |i| i.gsub("_", "-") }
 end
 
-locale_path = Rails.root.join("config/locales")
+old_debug, $DEBUG = $DEBUG, nil
```

Comment:
Not really.

Firstly, I'm really not sure [this change](https://github.com/ManageIQ/manageiq/pull/2295/files?w=1) is enough code for anything to be "a lot less".

Secondly, we're changing the `$DEBUG` flag, not a stream. If we just affected the stream, it'd still be going to the useless effort of producing a huge flood of output we're just going to ignore. And I wouldn't want to trample on any legitimate non-debug output/warnings/complaints.

Thirdly, `silence_stream` and friends have been deprecated & removed (in 4.2 and 5.0, respectively), because it's too tempting to use them in contexts where their lack of thread-safety becomes an issue.